packages/@angular/cli/utilities/is-directory.ts

```diff
@@ -0,0 +1,9 @@
+import * as fs from 'fs';
+
+export function isDirectory(path: string) {
+  try {
+    return fs.statSync(path).isDirectory();
+  } catch (_) {
```

Comment:
If TS is upgraded to 2.5.2 for CLI build purposes, the empty catch variable can be removed

Reply:
We should do that project wide when updating.